src/pygeoprocessing/geoprocessing.py

```diff
@@ -455,7 +460,7 @@ def raster_calculator(
                     target_block = target_block[target_block != nodata_target]
                 target_block = target_block.astype(numpy.float64).flatten()
 
-                if sys.version_info >= (3, 8):
+                if sys.version_info >= (3, 8) and use_shared_memory:
```

Comment:
Ah, yea I thought this was a Python 3.8 thing. Should it be mentioned in the API docs or HISTORY that this is for Python 3.8+ users? Speaking of which, Pyinstaller just released 4.1 which supports Python 3.8 and 3.9. So InVEST should officially support at least Python 3.8 very soon.

Reply:
Sure, added in both in 107cbfa0bbb0cb3ed4dabd7ffda6e611a2887f35